lib/rubocop/cop/rspec/repeated_include_example.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check for repeated include of shared examples.
+      #
+      # @example
+      #
+      #    # bad
+      #    describe 'foo' do
+      #      include_examples 'cool stuff'
+      #      include_examples 'cool stuff'
+      #    end
+      #
+      #    # bad
+      #    describe 'foo' do
+      #      it_behaves_like 'a cool', 'thing'
+      #      it_behaves_like 'a cool', 'thing'
+      #    end
+      #
+      #    # bad
+      #    context 'foo' do
+      #      it_should_behave_like 'a duck'
+      #      it_should_behave_like 'a duck'
+      #    end
+      #
+      #    # good
+      #    describe 'foo' do
+      #      include_examples 'cool stuff'
+      #    end
+      #
+      #    describe 'bar' do
+      #      include_examples 'cool stuff'
+      #    end
+      #
+      #    # good
+      #    describe 'foo' do
+      #      it_behaves_like 'a cool', 'thing'
+      #      it_bahaves_like 'a cool', 'person'
+      #    end
+      #
+      #    # good
+      #    context 'foo' do
+      #      it_should_behave_like 'a duck'
+      #      it_should_behave_like 'a goose'
+      #    end
+      #
+      #    # good
+      #    context 'foo' do
+      #      cool = 'neat'
+      #      include_examples 'something', cool
+      #      cool = 'fun'
+      #      include_examples 'something', cool
+      #    end
+      #
+      class RepeatedIncludeExample < Base
+        MSG = 'Repeated include of shared_examples %<name>s on line(s) %<loc>s'
+
+        def_node_matcher :several_include_examples?, <<-PATTERN
+          (begin <#include_examples? #include_examples? ...>)
```

Comment:
Will it match if there's something in between, e.g.
```ruby
context 'foo' do
  it_should_behave_like 'a duck'
  it {}
  it_should_behave_like 'a duck'
end
```

Reply:
Yes it will, I'll make sure there's a test case to indicate that.

Reply:
Tested, this works fine. Maybe we don't need an example, we would be testing the node pattern children matching.